src/acquisition/covid_hosp/state_timeseries/database.py

```diff
@@ -225,5 +225,4 @@ def __init__(self, *args, **kwargs):
         table_name=Database.TABLE_NAME,
         hhs_dataset_id=Network.DATASET_ID,
         columns_and_types=Database.ORDERED_CSV_COLUMNS,
-        key_columns=Database.KEY_COLS,
-        additional_fields=[Columndef('T', 'record_type', None)])
```

Comment:
please also remove `additional_fields` from the rest of `src/acquisition/covid_hosp/*`, as nothing else uses it now and it will only serve to add complexity and confusion.